.codeclimate.yml

```diff
@@ -8,6 +8,16 @@ engines:
     checks:
       overqualified-elements:
         enabled: false
+      order-alphabetical:
+        enabled: false
+      adjoining-classes:
```

Comment:
Agreed. These are used everywhere in 7.x. 
What do others think about 8.x? BEM conventions suggest we need to avoid this as well.